source/parse.h

```diff
@@ -7266,7 +7266,8 @@ class parser
     auto type_id(
         bool allow_omitting_type_name = false,
         bool allow_constraint         = false,
-        bool allow_function_type      = false
+        bool allow_function_type      = false,
+        bool allow_qualified_function_type = true
```

Comment:
This new parameter doesn't seem to be used - it always has its default value. Should it just be removed?